src/onnx/parse_qlinearconv.cpp

```diff
@@ -0,0 +1,277 @@
+            auto auto_pad = info.attributes.at("auto_pad").s();
+            if(auto_pad.find("NOTSET") == std::string::npos)
+                MIGRAPHX_THROW("QLINEARCONV: auto_pad option fix..");
+        }
+
+        if(contains(info.attributes, "group"))
+            values["group"] = parser.parse_value(info.attributes.at("group")).template at<int>();
+
+        if(not values.empty())
+            recalc_conv_attributes(values, kdims);
+
+        return values;
+    }
+
+    // This method is to prep for quantizelinear or dequantizelinear operation for
+    // either the broadcasting of weight-scale or zero-points of qlinearconv operator
+    // outputs: operator op (inputs x, broadcasted: scale (float) & zero_pt (8-bit))
+    instruction_ref bcast_qdq_instr(const std::string& op_name,
+                                    const instruction_ref x_in,
+                                    const instruction_ref arg_fscale,
+                                    const instruction_ref arg_z_pt,
+                                    const onnx_parser::node_info& info) const
+    {
+        auto in_lens = x_in->get_shape().lens();
+
+        // prep 1: broadcast scale. it can come as a scalar or a 1-D tensor.
+        instruction_ref bcast_scale;
+        if(arg_fscale->get_shape().elements() > 1)
+            bcast_scale = info.add_instruction(
+                migraphx::make_op("broadcast", {{"axis", 0}, {"out_lens", in_lens}}), arg_fscale);
+        else
+            bcast_scale = info.add_instruction(
+                migraphx::make_op("multibroadcast", {{"out_lens", in_lens}}), arg_fscale);
+
+        // prep 2: broadcast zero point. it can come as a scalar or a 1-D tensor.
+        instruction_ref bcast_zero_pt;
+        if(arg_z_pt->get_shape().elements() > 1)
+            bcast_zero_pt = info.add_instruction(
+                migraphx::make_op("broadcast", {{"axis", 0}, {"out_lens", in_lens}}), arg_z_pt);
+        else
+            bcast_zero_pt = info.add_instruction(
+                migraphx::make_op("multibroadcast", {{"out_lens", in_lens}}), arg_z_pt);
+
+        // op_name is either quantizelinear or dequantizelinear:
+        return info.add_instruction(migraphx::make_op(op_name), x_in, bcast_scale, bcast_zero_pt);
+    }
+
+    instruction_ref add_bias_to_conv(const instruction_ref bias_arg,
+                                     const instruction_ref conv_instr,
+                                     const onnx_parser::node_info& info) const
+    {
+        auto conv_sh   = conv_instr->get_shape();
+        auto conv_lens = conv_sh.lens();
+        auto conv_type = conv_sh.type();
+
+        auto broadcast_bias = info.add_instruction(
+            migraphx::make_op("broadcast", {{"axis", 1}, {"out_lens", conv_lens}}), bias_arg);
+        auto f_bias =
+            info.add_instruction(make_op("convert", {{"target_type", conv_type}}), broadcast_bias);
+
+        return info.add_instruction(migraphx::make_op("add"), conv_instr, f_bias);
```

Comment:
This can make use of `add_common_op` which would handle broadcasting and conversion. 

Or better there is `add_bias` inside `onnx_parser.cpp` which can be modified to handle better handle quantized convolutions.

Reply:
(We had a parallel discussion in a different thread and won't be using add_common_op). Thanks.

Reply:
What was the reason why `add_common_op` can't be used ?. To me looks like this can still use `add_common_op` which. Doesn't make a difference though in terms of logic though.  Therefore minor point. 

FYI there is also `add_bias` inside `onnx_parser.cpp` which is better suited.

Reply:
add_bias(), as I see it is specific to 3D tensors. There is no such assumption here in this operator. Thanks.

Reply:
> add_bias(), as I see it is specific to 3D tensor 
where do see that ? i think it just checks for input arg sizes to be 3

Reply:
Ah, sorry, I misread it.. Thanks.

I am looking inside it carefully. I have a convert operation in place here.. that is missing there..